### PR TITLE
Docs/refactors configure panels

### PR DESCRIPTION
**Pull request by chri2547**
This PR refactors the configure panel options content into a single topic.

### CODE REVIEW

File: docs/sources/panels/configure-panel-options/index.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+A Grafana panel is the user interface you use to define a data source query, and transform and format data that appears in visualizations.
+
+A panel editor includes a query builder and a series of options that you can use to transform data and add information to your panels.
+
+This topic describes how to:
+
+- open a panel for editing
+- add a panel title and description
+- view a panel JSON model
+- add repeating rows and panels
```
Comment (torkelo) [inline docs/sources/panels/configure-panel-options/index.md:35]: Should these not be capitalized?

File: docs/sources/panels/configure-panel-options/index.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+After you add a panel to a dashboard, you can open it at any time to change change or update queries, add data transformation, and change visualization settings.
+
+1. Open the dashboard that contains the panel you want to edit.
+
+1. Click in the panel title and select **Edit**.
+
+   To use a keyboard shortcut to open the panel, hover over the panel and press `e`.
+
+   The panel opens in edit mode.
```
Comment (torkelo) [inline docs/sources/panels/configure-panel-options/index.md:47]: Nitpick, these empty lines between these sentences look a bit strange not sure if they are intentional or not :shrug:  (not very important, just found it curious) 
Screenshot from 2022-08-10 11-45-03

File: docs/sources/panels/configure-panel-options/index.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+1. On the dashboard home page, click **Add panel**.
+
+1. On the **Add a panel** dialog box, click **Add a new row**.
+
+1. Hover over the row title and click the cog icon.
+
+1. On the **Row Options** dialog box, add a title and select the variable for which you want to add repeating rows.
+
+> **Note:** To provide context to dashboard users, add the variable to the row title.
```
Comment (torkelo) [inline docs/sources/panels/configure-panel-options/index.md:117]: Configure repeating rows feels like it does not belong to this topic, but more a "build and organize a dashboard" topic, like this one: 
http://localhost:3002/docs/grafana/next/dashboards/add-organize-panels/

**Review by torkelo — APPROVED** [review]
Looks good, just one minor comment on repeating rows that feels out of place

File: docs/sources/panels/configure-panel-options/index.md (hunk 2 above)
Comment (chri2547) [reply docs/sources/panels/configure-panel-options/index.md:47]: Yeah, I agree. I can fix that. Thanks.

Comment (chri2547) [reply docs/sources/panels/configure-panel-options/index.md]: Done.

File: docs/sources/panels/configure-panel-options/index.md (hunk 1 above)
Comment (chri2547) [reply docs/sources/panels/configure-panel-options/index.md]: Done.

**Review by torkelo — APPROVED** [review]

**Pull request merged**